Remove endpoint list and add info about the default user credentials (#1048)
spryker/spryker-docs: docs/scos/dev/setup/installing-spryker-with-docker/installation-guides/installing-in-demo-mode-on-macos-and-linux.md

3. Update the `hosts` file:

Follow the installation instructions in the white box from the `docker/sdk bootstrap` command execution results to prepare the environment.

{% info_block infoBox %}

>  You can run `docker/sdk install` after `bootstrap` to get the list of the instructions.

Reviewer: ```suggestion
To get the list of the instructions, you can run `docker/sdk install` after `bootstrap`.
```

{% endinfo_block %}

{% info_block warningBox %}

> Depending on the hardware performance, the first project launch can take up to **20 minutes**.

Reviewer: ```suggestion
Depending on the hardware performance, the first project launch can take up to *20 minutes*.
```

{% endinfo_block %}

## Endpoints

To ensure that the installation is successful, make sure you can access the configured endpoints from the Deploy file. For more information about the Deploy file, see [Deploy file reference - 1.0](/docs/scos/dev/the-docker-sdk/{{site.version}}/deploy-file/deploy-file-reference-1.0.html).

### Back-Office

The default credentials to access the Back Office are located inside `/src/Pyz/Zed/User/UserConfig.php`.

### RabbitMQ

> To access RabbitMQ UI, use `spryker` as a username and `secret` as a password. You can adjust the credentials in `deploy.yml`. See [Deploy file reference - 1.0](/docs/scos/dev/the-docker-sdk/{{site.version}}/deploy-file/deploy-file-reference-1.0.html) to learn about the Deploy file.

Reviewer: ```suggestion
To access RabbitMQ UI, use `spryker` as a username and `secret` as a password. You can adjust the credentials in `deploy.yml`. For information about the Deploy file, see [Deploy file reference - 1.0](/docs/scos/dev/the-docker-sdk/{{site.version}}/deploy-file/deploy-file-reference-1.0.html).
```
